pkg/e2e/operators/cloudingress/installed.go

```diff
@@ -1,31 +1,24 @@
 package cloudingress
 
 import (
-	"context"
-
 	"github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 
 	"github.com/openshift/osde2e/pkg/common/helper"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // tests
+
+func checkDeployment(h *helper.H, namespace string, name string, defaultDesiredReplicas int32)
+
```

Comment:
what's going on here? this is go syntax i'm not familiar with

Reply:
That's an error on my part.